src/source/Rtp/RtpPacket.c

```diff
@@ -75,7 +75,7 @@ STATUS freeRtpPacket(PRtpPacket * ppRtpPacket)
     // freeRtpPacket is idempotent
     CHK(*ppRtpPacket != NULL, retStatus);
 
-    MEMFREE(*ppRtpPacket);
+    SAFE_MEMFREE(*ppRtpPacket);
 CleanUp:
     CHK_LOG_ERR(retStatus);
 
```

Comment:
For some reason this tool doesn't let me comment on the other lines. 
In the next function - createRtpPacketFromBytes - when MEMALLOCing - cast the result and check for NULL